UIKit

Dante13Dark opened the pull request:
GitHubPages: https://dante13dark.github.io/lessons-JS-core/UIKit/UIKit.html

Dante13Dark commented:
прогнал код через prettier

Dante13Dark committed:
UIKit

Muslik commented on UIKit/UIKit.html:
```diff
@@ -0,0 +1,715 @@
+      <div class="buttons dashed-wrapper">
+        <div class="buttons__section">
+          <button class="button button_medium button_color_primary">
+            <div class="button__icon-container">
```
Зачем нужна лишняя обертка

Dante13Dark replied:
поправил цвета у searchbar, убрал лишние контейнеры у иконок, чекбокса, радио и у них же в дропдаунах, растянул их на всю ячейку списка

Muslik reviewed (changes requested):
- цвет placeholder в searchbar не как на макете
- radio, checkbox в dropdown должны быть по высоте и длине как ```li```

Dante13Dark committed:
Правки замечаний

поправил цвета у searchbar, убрал лишние контейнеры у иконок, чекбокса, радио и у них же в дропдаунах
